### PR TITLE
Ripanga Restyle

### PR DESCRIPTION
**Pull request by piercefrawley**
screen shot 2017-06-22 at 3 41 47 pm

### CODE REVIEW

File: tables/ripanga/sandbox/sandbox.jsx [hunk 1]
```diff
@@ -90,7 +69,24 @@ render(
       Uncheck All
     </button>
 
-    {groupedWithCheckboxes}
+    <Ripanga
+      showCheckboxes={true}
```
Comment (ben-burlingham) [inline tables/ripanga/sandbox/sandbox.jsx]: Oh you

File: tables/ripanga/sandbox/sandbox.jsx [hunk 2]
```diff
@@ -90,7 +69,24 @@ render(
...
+      globalKey='ripanga-sandbox'
+      idKey='key'
+      scope='sandbox'
+      onMounted={state => console.log(state)}
```
Comment (ben-burlingham) [inline tables/ripanga/sandbox/sandbox.jsx]: shut up shut up shut up shut up

Comment (ben-burlingham) [inline tables/ripanga/sandbox/sandbox.jsx]: nb Could pass a function reference here if you like: `onMounted={console.log}`

File: tables/ripanga/src/Ripanga.scss [hunk 3]
```diff
@@ -67,16 +67,11 @@ $tableZ: 0;
...
-  :global .fa {
-    font-size: 12px;
-    margin-left: 5px;
-  }
 }
 
 .headCell {
   composes: tableCell;
-  border-width: 0 1px 0 0;
+  border-width: 0 0 0 0;
```
Comment (chrisarevir) [inline tables/ripanga/src/Ripanga.scss]: can this just be `0` then?

File: tables/ripanga/src/Ripanga.scss [hunk 4]
```diff
@@ -96,20 +91,19 @@ $tableZ: 0;
...
-  text-align: center;
+  justify-content: center;
+  max-width: 40 + (2 * $padding) + 2;
+  min-width: 40 + (2 * $padding) + 2;
 }
 
 .controlHeadCell {
   composes: controlCell;
-  border-width: 0 1px 0 0;
+  border-width: 0 0 0 0;
```
Comment (chrisarevir) [inline tables/ripanga/src/Ripanga.scss]: same as above

**Comment by ben-burlingham** [comment]
Also aren't these just the changes I made - you did a bunch as well, right?

**Pull request merged**